flask/app/participants.py

```diff
@@ -37,22 +37,36 @@
 def list_participants():
 
     # parsing query parameters
-    max_rows = request.args.get("max_rows", default=100)
+    limit = request.args.get("limit", default=10)
+    page = request.args.get("page", default=1)
     starts_with = request.args.get("starts_with", default="", type=str)
     starts_with = f"{starts_with}%"  # sql syntax
     order_by_col = request.args.get("order", default="participant_id", type=str)
-    # need some default or we need an ifelse statement, one with order_by method and one without. AFAIK there is no 'default' parameter we can pass into order_by to get the default sql ordering scheme
+    filter_by = request.args.get("filter", default="participant_id,", type=str)
+    filter_by_col, filter_val = filter_by.split(",")[0], filter_by.split(",")[1]
```

Comment:
You'll get an IndexError here if there's no comma. Is this meant to be a basis to support multiple filters?

Reply:
No, just to combine the query parameters for the filter column and filter value. Instead, I can separate these into two query parameters, e.g. filter_column and filter_value.

Reply:
fixed in 920f9b2c23911b11e012ee51312d3751d1142d33